README.md

```diff
@@ -123,7 +130,14 @@ docker run -d --name incusredis redis
 Start Incus:
 
 ```Shell
-docker run -d --link incusredis:redis  --name incus \
+docker run --env-file incus.conf -d --link incusredis:redis  --name incus \
+        -e REDIS_ENABLED=true -p 4000:4000 jwgur/incus 
```

Comment:
Instead of using jwgur/incus we should include a build step `docker build -t imgur/incus .`